README.rst

```diff
@@ -145,3 +147,11 @@ simply be appended to the invocation as follows::
     python3 -m pymongoexplain <path/to/your/script.py> [PARAMS] [--optname OPTS]
 
 
+Limitations
+-----------
+This package does not support the fluent `Cursor API <https://api.mongodb.com/python/current/api/pymongo/cursor.html>`_,
```

Comment:
This is not the correct link for PyMongo documentation. It should be: https://pymongo.readthedocs.io/en/stable/api/pymongo/cursor.html

Reply:
Done.